src/utils/barChart.utils.ts

```diff
@@ -10,7 +10,7 @@ export const barsWidgetsLabels: Record<string, Array<string>> = {
     ['textView.fatal.plural', 'textView.severe.plural', 'textView.light.plural'],
   'injured_count_by_accident_year':
     ['textView.killed.plural', 'textView.severeInjured.plural', 'textView.lightInjured.plural'],
-}
+  }
```

Comment:
I suppose this changes happed automatically in your editor, but you shouldn't push them